examples/GainPlugin/Source/PluginEditor.cpp

```diff
@@ -81,35 +81,35 @@ GainPluginAudioProcessorEditor::GainPluginAudioProcessorEditor (GainPluginAudioP
 
     // Lastly, start our timer for reporting meter values
     startTimerHz(30);
-
+
```

Comment:
Apologies @nick-thompson, not sure what happened here. I jumped to mac to check example apps were working (have been working on Linux) and I think my editor must have converted some line endings.

Reply:
No worries! This kinda thing would be solved by #24 anyway... getting some kind of clang-format thing going so that anybody who wants to contribute doesn't have to worry about manally upholding any style guide or manually change their editor line endings or anything. Just clang-format before merge && :+1:. But until we have that I'm quite relaxed about things like this!